setup.cfg

```diff
@@ -1,6 +1,3 @@
-[bdist_wheel]
-universal = 0
-
```

Comment:
Are you sure about this? It was added to prevent creating wheels which claim Python 2 support in their name.

Reply:
0 is the default.

To confirm, here's the result of `python -m build` on `master`:

```
pytest-rerunfailures-13.1.dev0.tar.gz           pytest_rerunfailures-13.1.dev0-py3-none-any.whl
```

and with this PR:

```
pytest-rerunfailures-13.1.dev0.tar.gz           pytest_rerunfailures-13.1.dev0-py3-none-any.whl
```

Neither has `py2` in the filename.

Reply:
That's nice to hear, so we can remove this setting from all other repos, too. Thank you.

Reply:
Tiny PRs, or roll it into other updates (for example, removing 3.7, or adding 3.12) :)

Reply:
Happy to help if you tell me which projects...